lib/fizzy/execute.cpp

```diff
@@ -1226,12 +1288,12 @@ ExecutionResult execute(Instance& instance, FuncIdx func_idx, const Value* args,
         case Instr::f32_abs:
         {
             // TODO: This can be optimized https://godbolt.org/z/aPqvfo
```

Comment:
Please also remove all these TODO comments where the implementation was changed, because the comments are not relevant anymore.